package.json

```diff
@@ -33,15 +33,15 @@
   },
   "homepage": "https://github.com/indutny/llparse-test-fixture#readme",
   "devDependencies": {
-    "@types/mocha": "^5.2.7",
-    "@types/node": "^10.17.27",
-    "mocha": "^7.1.1",
-    "ts-node": "^8.8.1",
-    "tslint": "^5.20.1",
-    "typescript": "^3.9.6"
+    "@stylistic/eslint-plugin": "^1.5.4",
+    "@types/node": "^20.11.16",
+    "@typescript-eslint/eslint-plugin": "^6.20.0",
+    "@typescript-eslint/parser": "^6.20.0",
+    "eslint": "^8.56.0",
+    "ts-node": "^10.9.2",
```

Comment:
nit: We can drop ts-node and use `--experimental-strip-types` (and still keep typescript in order to have type checking)

Reply:
I would like, but I cant use  --experimental-strip-types with latest node 22 which is available via nvm.